urdf_mesh_converter/README.md

```diff
@@ -0,0 +1,21 @@
+# URDF Mesh Converter
+
+This package leverages the URDF Exporter plugin for SolidWorks to generate colored `.dae` models from `.3dxml` files.
```

Comment:
I think this sentence is a little bit misleading.
This package itself does not use URDF Exporter.
How about something like
```
This package converts the colored output of the URDF Exporter plugin of SolidWorks into the format readable by ROS systems.
Specifically, this package generates colored `.dae` models from `.3dxml` files.
```
?

Reply:
Thanks.
I changed your sentence because it is easier to understand.
https://github.com/jsk-ros-pkg/jsk_model_tools/pull/255/commits/c5b5ee03bf3485147673fa7e64fd3fc26227e553